addOns/automation/CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this add-on will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## Unreleased
+### Fixed
+- Added fix for resolving fileName in import job when fileName is provided in an environment variable
```

Comment:
e.g.`- Correct file path resolution with environment variables.` this applies to all jobs that make use of files not just import (e.g. OpenAPI).